scripts/apps/authoring/authoring/components/line-count.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {get} from 'lodash';
+import {stripHtmlTags, gettextPlural} from 'core/utils';
+
+interface IProps {
+    item: string;
+    config: {
```

Comment:
Import config(from `scripts/appConfig.ts`) instead of connecting a service. Connecting services is a crutch we use to get angular services, but the longer I think about it, the more I'm convinced that angular services are better of being standalone es6 modules. We would avoid mountains of boilerplate and wouldn't have to deal with async stuff when it's actually synchronous. What do you think?

Reply:
we merge that config from core with superdesk repo config, so I can't just import it I think

Reply:
You can. It's merged at build, not at runtime https://github.com/superdesk/superdesk-client-core/blob/master/webpack.config.js#L62